>>>> [1] invert1 <<<<
Executing command 'open' ...
Executing command 'invert' ...
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 165020) >= this->size() (which is 165020)
>>>> [2] invert2 <<<<
Executing command 'open' ...
Executing command 'invert' ...
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 165020) >= this->size() (which is 165020)
>>>> [3] invert3 <<<<
Executing command 'open' ...
Executing command 'invert' ...
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 99170) >= this->size() (which is 99170)
>>>> [4] invert4 <<<<
Executing command 'open' ...
Executing command 'invert' ...
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 617600) >= this->size() (which is 617600)
